CMakeLists.txt

```diff
@@ -136,12 +181,13 @@ macro(xeus_r_set_common_options target_name)
         target_compile_options(${target_name} PUBLIC -Wunused-parameter -Wextra -Wreorder)
 
         # C++17 flag
-        CHECK_CXX_COMPILER_FLAG("-std=c++17" HAS_CPP_17_FLAG)
-        if (HAS_CPP_17_FLAG)
-            target_compile_features(${target_name} PRIVATE cxx_std_17)
-        else ()
-            message(FATAL_ERROR "Unsupported compiler -- xeus-r requires C++17 support!")
-        endif ()
+        target_compile_features(${target_name} PRIVATE cxx_std_17)
+        # CHECK_CXX_COMPILER_FLAG("-std=c++17" HAS_CPP_17_FLAG)
```

Comment:
This fails because of the `-Wl,-v` flags mentioned in the other comment. If we don't pass those flags when building with emscripten, this check should pass again.

Reply:
Yes as I replied above, this flag doesn't look like we need it. Thanks for pointing it out.